app/src/main/java/org/cru/godtools/ui/tools/ToolsCategoryFragment.kt

```diff
@@ -0,0 +1,9 @@
+package org.cru.godtools.ui.tools
```

Comment:
put all of this in the `org.cru.godtools.ui.dashboard.tools` package

Reply:
@frett did you mean `org.cru.godtools.base.ui.dashboard`.

Reply:
no, as a sub package of this: https://github.com/CruGlobal/godtools-android/tree/develop/app/src/main/java/org/cru/godtools/ui/dashboard

Reply:
so, keep it in the `app` module, just move the java package under the `ui/dashboard` package that already exists

Reply:
OK I see thanks for the clarification